crates/test_utils/src/connector_auth.rs

```diff
@@ -326,6 +376,10 @@ pub enum ConnectorAuthType {
         api_secret: Secret<String>,
         key2: Secret<String>,
     },
+    PayoutKeys {
+        adyen: SignatureKey,
+        wise: BodyKey,
+    },
```

Comment:
I think we could achieve this by tweaking around the `pre_request.js` script in MCA step. Something like:
```txt
|-MCA_dir
    |- MCA_adyen
    \- MCA_wise
```

pass the auth for both in a single go, and then, extract?

Also, Adyen and Wise are 2 different connectors, can we have 2 different collections by separating them? This will greatly reduce complexity from what I understand